src/screens/Home/Home.js

```diff
@@ -1,6 +1,39 @@
-import React from 'react';
-import { Hero } from '@/components';
+import React, { useEffect, useState } from 'react';
+import { Dimensions, View, FlatList } from 'react-native';
+import { Hero, MovieList } from '@/components';
+import { getTrending, getRecentlyAdded } from '@/controllers/MovieController';
 
 export function Home() {
-  return <Hero />;
+  const [movieLists, setMovieLists] = useState([]);
+
+  const windowHeight = Dimensions.get('window').height;
+
+  useEffect(() => {
+    Promise.all([getTrending(), getRecentlyAdded()])
+      .then((values) => {
+        const myList = values[0].results.slice(10, 20);
+        const trendingNow = values[0].results.slice(0, 10);
+        const recentlyAdded = values[1].results.slice(0, 10);
+        setMovieLists([
+          { title: 'My List', movieList: myList },
+          { title: 'Trending Now', movieList: trendingNow },
+          { title: 'Recently Added', movieList: recentlyAdded },
+        ]);
+      })
+      .catch((error) => console.log(error));
```

Comment:
this is fine, but we try to use async await. 
We can use await Promise.all with a try and catch.

Also, for the titles you should use the language file